### PR TITLE
TheHive Cortext Analyzer documentation

### PR DESCRIPTION
**Pull request by rr404**

### CODE REVIEW

File: crowdsec-docs/docs/cti_api/integration_thehive.md [hunk 1]
```diff
@@ -1,5 +1,44 @@
...
 sidebar_position: 2
 ---
+
+A **TheHive - Cortext Analyzer** that allows you obtain a detailed report from CrowdSec's CTI **smoke** database
+
+https://github.com/TheHive-Project/Cortex-Analyzers/tree/develop/analyzers/Crowdsec
+
+## Installation
+
+The CrowdSec analyzer and its report template will soon be included in TheHive
```
[the remark this answers is not in the document]
Comment (rr404) [reply crowdsec-docs/docs/cti_api/integration_thehive.md]: OK added full stops  to the end of sentenses

File: crowdsec-docs/docs/cti_api/integration_thehive.md [hunk 2]
```diff
@@ -1,5 +1,44 @@
...
+
+The development version can be found here:
+* [develop branch analyzer](https://github.com/TheHive-Project/Cortex-Analyzers/tree/develop/analyzers/Crowdsec
+)
+* [develop branch template](https://github.com/TheHive-Project/Cortex-Analyzers/tree/develop/thehive-templates/Crowdsec_1_0)
+
+To add the analyzer to a case observable you can refer to the [offical doc](https://docs.thehive-project.org/thehive/user-guides/analysts/run-analyzers/)
+
+To complete/customize the template you refer to this [how to](https://docs.thehive-project.org/thehive/user-guides/administrators/analyzer-templates/)
```
[the remark this answers is not in the document]
Comment (rr404) [reply crowdsec-docs/docs/cti_api/integration_thehive.md]: done

File: crowdsec-docs/docs/cti_api/integration_thehive.md [hunk 3]
```diff
@@ -1,5 +1,44 @@
...
+## Installation
+
+The CrowdSec analyzer and its report template will soon be included in TheHive
+
+The development version can be found here:
+* [develop branch analyzer](https://github.com/TheHive-Project/Cortex-Analyzers/tree/develop/analyzers/Crowdsec
+)
+* [develop branch template](https://github.com/TheHive-Project/Cortex-Analyzers/tree/develop/thehive-templates/Crowdsec_1_0)
+
+To add the analyzer to a case observable you can refer to the [offical doc](https://docs.thehive-project.org/thehive/user-guides/analysts/run-analyzers/)
```
[the remark this answers is not in the document]
Comment (rr404) [reply crowdsec-docs/docs/cti_api/integration_thehive.md]: ok

**Review by AlteredCoder — APPROVED** [review]
lgtm

**Pull request merged**